model/scan.py

```diff
@@ -154,6 +154,9 @@ def single_file(filename, exposure_times, osc_start, osc_width, epoch):
         index = scan_helper_image_files.image_to_index(os.path.split(filename)[-1])
         if epoch is None:
             epoch = 0.0
+        if osc_width < 0:
+            osc_width += 360
```

Comment:
This looks funny, because `osc_width` is usually interpreted as the `"Angle_increment"`, and adding 360 to this does not seem right. See e.g.
https://github.com/cctbx/dxtbx/blob/2cb9ef2819ea43ba07d4ef34576382dfef596c98/format/FormatCBFMini.py#L215-L227

Reply:
The case I was capturing here was the width being -359° as in the user provided example because the image went from 359° to 0°

I _could_ express this as e.g. if < -180° or something in the code? Would that be clearer?

Reply:
I've made that change and added some long form commentary above to indicate the problem this is trying to solve.

Reply:
Thanks, the additional comment makes sense